libraries/CurieBLE/examples/IMUBleNotification/IMUBleNotification.ino

```diff
@@ -0,0 +1,120 @@
+/*
+   Copyright (c) 2016 Intel Corporation.  All rights reserved.
+
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2.1 of the License, or (at your option) any later version.
+
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; if not, write to the Free Software
+   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+#include <CurieBLE.h>
+#include <CurieIMU.h>
+
+/*
+   This sketch example partially implements the standard Bluetooth Low-Energy Battery service.
```

Comment:
It looks the same... I'm talking about this line
`This sketch example partially implements the standard Bluetooth Low-Energy Battery service`

It's the same line from the BatteryMonitor sketch. Can you replace it with something that explains to the user what the sketch is doing? and likewise for the other examples?